scripts/jenkins-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+[ -d venv ] || virtualenv venv
+pip install -e .
+
+if [ "$ENVIRONMENT" = "Select one" -o "$ENVIRONMENT" = "" ]; then
```

Comment:
That's a good point. I'm not sure if Jenkins is the place we need to be worried about being commonly understood since it's less accessible than the deployment repository itself. At the same time, I feel like it's best to be consistent in our usage of `stage` vs `environment`, otherwise it can get confusing.

Maybe it's worth changing `stage` to `environment` and `environment` to `??` since this keeps coming up.